actions.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+generate-certificate:
+  description: Generate a certificate along with private key.
+  params:
+    cn:
```

Comment:
`common-name` would be better then `cn`. Let's avoid acronyms when possible.

Reply:
Done